CMakeLists.txt

```
@@ -138,6 +138,13 @@ if (AMR_WIND_ENABLE_OPENFAST)
   if(OpenFAST_FOUND)
     message(STATUS "Found OpenFAST = ${OPENFAST_DIR}")
   endif()
+  include(CheckIncludeFileCXX)
+  set(CMAKE_REQUIRED_INCLUDES "${OpenFAST_INCLUDE_DIRS}")
+  check_include_file_cxx("SCDataEx_Types.h" USE_SCDX)
+  message(STATUS "AMR_WIND_FAST_USE_SCDX is: ${AMR_WIND_FAST_USE_SCDX}")
+  if(USE_SCDX)
+    add_definitions(-DAMR_WIND_FAST_USE_SCDX=1)
```

```suggestion
    target_compile_definitions(${amr_wind_lib_name} PUBLIC AMR_WIND_FAST_USE_SCDX)
```

To be consistent with modern CMake usage (see L148). This will export the compile definitions as part of the AMR-Wind target if someone wants to use AMR-Wind as a library with other codes.